lesson11/index11.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Burger_Rectaurant</title>
+        <link rel="stylesheet" href="style11.css">
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+        <link href="https://fonts.googleapis.com/css2?family=Raleway&display=swap" rel="stylesheet">
+    </head>
+    <body>
+        <nav>
+           <a href="">Home</a>
```

Comment:
put # everywhere in href for now

without it, browser will not consider this button as a link